Support aten::leaky_relu for TE

edvgha opened the pull request:
Test Plan:
./bin/test_tensorexpr

python test/test_jit_fuser_te.py TestTEFuser.test_unary_ops

Differential Revision: D28499776

edvgha commented:
there is dependency from #58337

Krovatkin reviewed (approved):
:shipit: looks reasonable to me

edvgha committed:
Support aten::leaky_relu for TE (#58464)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/58464

Test Plan:
./bin/test_tensorexpr

python test/test_jit_fuser_te.py TestTEFuser.test_unary_ops

Reviewed By: Krovatkin

Differential Revision: D28499776

fbshipit-source-id: 86212b6385fddeda1b68ff2e8390462110d1ee69
